python/pyspark/sql/functions.py

```diff
@@ -1077,6 +1077,35 @@ def pow(col1: Union["ColumnOrName", float], col2: Union["ColumnOrName", float])
     return _invoke_binary_math_function("pow", col1, col2)
 
 
+def pmod(dividend: Union["ColumnOrName", float], divisor: Union["ColumnOrName", float]) -> Column:
+    """
+    Returns the positive value of dividend mod divisor.
+
+    .. versionadded:: 3.4.0
+
+    Examples
+    --------
+    >>> df = spark.createDataFrame([
+    ...     (1.0, float('nan')), (float('nan'), 2.0),
+    ...     (float('nan'), float('nan')), (-3.0, 4.0),
+    ...      (-5.0, -6.0), (7.0, -8.0), (1.0, 2.0)],
```

Comment:
These are all good examples but it would be good to also have examples where |dividend| > divisor > 0 as that is normal usage of this function.

Reply:
done. thanks!